src/system/dev/vfs.c

```diff
@@ -89,19 +89,29 @@ int vfs_update_entry(int file, struct fs_driver const* const driver, void* arg)
 	return 0;
 }
 
-int _open(const char* file, int flags, int mode) {
-	struct _reent* r = _REENT;
-	// Check if the filename is too long or not NULL terminated
+bool file_size(const char* file) {
+	// Returns true if is under size constraint
 	size_t i = 0;
 	for (i = 0; i < MAX_FILELEN; i++) {
 		if (file[i] == '\0') {
 			break;
 		}
 	}
 	if (i == MAX_FILELEN) {
+		return i != MAX_FILELEN;
+	} else {
+		return true;
+	}
```

Comment:
As Alex was saying, I was trying to say that this can be condensed down to return i != MAX_FILELEN;

Reply:
:+1: Done